package.json

```
@@ -2,30 +2,45 @@
   "name": "grenoble-hands-on-react",
   "version": "0.1.0",
   "private": true,
-  "dependencies": {
-    "@testing-library/jest-dom": "^5.11.4",
-    "@testing-library/react": "^11.1.0",
-    "@testing-library/user-event": "^12.1.10",
-    "bulma": "^0.9.1",
-    "history": "^5.0.0",
-    "leaflet": "^1.7.1",
-    "react": "^17.0.1",
-    "react-dom": "^17.0.1",
-    "react-router": "^6.0.0-beta.0",
-    "react-router-dom": "6.0.0-beta.0",
-    "react-scripts": "4.0.0"
+  "engines": {
+    "node": ">= 20",
+    "pnpm": ">= 8"
   },
+  "type": "module",
   "scripts": {
-    "start": "react-scripts start",
-    "build": "react-scripts build",
-    "test": "react-scripts test",
-    "eject": "react-scripts eject"
+    "dev": "vite",
+    "build": "vite build",
+    "lint": "eslint . --ext js,jsx --report-unused-disable-directives --max-warnings 0",
+    "preview": "vite preview",
+    "test": "vitest",
+    "prettier": "pnpm exec prettier . --write"
```

```suggestion
    "prettier": "prettier . --write"
```

ça ne marche pas ça ?